### PR TITLE
Add environment domains for all environments

### PR DESCRIPTION
**Pull request by johnake**
### Context

 Create find-teacher-training-courses.service.gov.uk domain
 Hosted zone and frontdoor already created

### Changes proposed in this pull request

 Create environment domains for all environments

### Guidance to review

 Run `make find <env> domains-plan`

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally
- [ ] Inform data insights team due to database changes

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -61,13 +61,16 @@ dv_review_aks: ## make dv_review_aks deploy APP_NAME=2222 CLUSTER=cluster1 USE_D
 
 qa_aks:
 	$(eval include global_config/qa_aks.sh)
+	$(eval DEPLOY_ENV=qa)
```
Comment (saliceti) [inline Makefile:64]: DEPLOY_ENV is already defined in the global_config eg: https://github.com/DFE-Digital/publish-teacher-training/blob/main/global_config/qa_aks.sh#L2
Let's not redefine it as it's confusing, but make it work with the value from global_config (ie rename your files)

File: terraform/custom_domains/environment_domains/workspace_variables/find_production_aks.tfvars.json [hunk 2]
```diff
@@ -12,6 +12,19 @@
...
+      "resource_group_name": "s189p01-fttdomains-rg",
+      "domains": [
+        "www"
+      ],
+      "cached_paths": [
+        "/assets/*"
+      ],
+      "environment_short": "pd",
+      "origin_hostname": "publish-production.teacherservices.cloud",
+      "null_host_header": true
```
Comment (saliceti) [inline terraform/custom_domains/environment_domains/workspace_variables/find_production_aks.tfvars.json:27]: If true, then we need to add a new ingress *find-teacher-training-courses.service.gov.uk*
Can we try without it so it uses *publish-production.teacherservices.cloud*?

**Comment by johnake** [comment]
[Create find-teacher-training-courses.service.gov.uk domain](https://trello.com/c/H7vjsEwQ/1663-create-find-teacher-training-coursesservicegovuk-domain)

**Review by saliceti — APPROVED** [review]

**Comment by saliceti** [comment]
Approved but converted to draft while we're waiting for:
- DNS delegation
- Rails routing

**Pull request merged**